students/NatalieRodriguez/Lesson02/grid_3_inputs.py

```diff
@@ -0,0 +1,25 @@
+def row(a):
+    print("+", end=" ")
+    for i in range(a-1):
+        print("- "*c + "+", end=" ")
+    print("- "*c + "+" )
+
+def column(a):
+    print("|", end=" ")
+    for i in range(a-1):
+        print(" "*(c*2) + "|", end=" ")
+    print(" "*(c*2) + "|" )
+
+def func(a,b):
+    row(a)
+    for i in range(b):
+        for i in range(c):
+            column(a)
+        row(a)
+
+a=int(input("Columns: "))
+b=int(input("Rows: "))
+c=int(input("Insides: "))
+func(a,b)
```

Comment:
need a bit more spacing:
```
a = int(input("Columns: "))
b = int(input("Rows: "))
c = int(input("Insides: "))
func(a, b)
```

also choose more descriptive var names when appropriate 🙂 

for example
```
column_size = int(input("Columns: "))
row_size = int(input("Rows: "))
insides_size = int(input("Insides: "))
func(column_size, row_size)

def func(column_size, row_size)
```

this is a small piece of code, but imagine lots of code and you're trying to step through to understand the logic.. then you go "what was 'a' var again?"